### PR TITLE
add PDR and ACS for shadow networks PoC

### PR DESCRIPTION
**Pull request by xh3b4sd**
This is our working copy of decisions and next steps. Let's collaborate and talk to get some shit done. 🚀

### CODE REVIEW

**Comment by marcusellison** [comment]
Hey @xh3b4sd I added in some thoughts and adjustments. Let me know if you're okay with changes or there's a place where we're not yet aligned.

File: acs/001-shadow-networks/spec.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+- An organization is e.g. Venturemark or Baronfig. An organization has
+  timelines.
+- A timeline is the place to write updates. A timeline is accessible for an
+  audience.
+- An audience is a group of users. For now we just have a few of them hardcoded,
+  e.g. customers, employees, investors and vendors.
+- An update is written text on a timeline's timeline. An update has comments and
+  replies.
+- A founder is the user-owner of the organization
+- A member is a user that has access to read/write timelines within an organization
```
Comment (xh3b4sd) [inline acs/001-shadow-networks/spec.md:25]: @marcusellison this is good to be aware of. Though this spec only aims to picture what we build now. If we say we want to have something within 7 days we are only building something that works locally. So there are no users and permissions.

File: acs/001-shadow-networks/spec.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+- As a founder I want to write updates myself. Choosing a timeline, I just write
+  my update in the text editor and add it to the timeline by clicking some
+  "send" button. If I try to add an update without choosing an audience there
+  will be an error displayed. I can choose multiple of the hard-coded audiences.
+  The new update gets added at the top of the list of updates I see. The red
+  notification dot does not occur for me if I add an update in this particular
+  timeline.
+
+- As a member of an organization, I can create an update for a timeline I follow.
+- As a member of an organization I can comment on an update.
```
Comment (xh3b4sd) [inline acs/001-shadow-networks/spec.md:67]: For the very next step this is too complicated as there will likely be no shared state. Later we need to work out how permissions should work.

File: acs/001-shadow-networks/spec.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+  updates on timelines are signified by some red notification dot on the
+  timeline within the sidebar.
+- As a founder I want to write updates myself. Choosing a timeline, I just write
+  my update in the text editor and add it to the timeline by clicking some
+  "send" button. If I try to add an update without choosing an audience there
+  will be an error displayed. I can choose multiple of the hard-coded audiences.
+  The new update gets added at the top of the list of updates I see. The red
+  notification dot does not occur for me if I add an update in this particular
+  timeline.
+
```
Comment (xh3b4sd) [inline acs/001-shadow-networks/spec.md:65]:
```suggestion
```

File: acs/001-shadow-networks/spec.md (hunk 1 above)
Comment (marcusellison) [reply acs/001-shadow-networks/spec.md:25]: Yup - good point here.

**Review by marcusellison — APPROVED** [review]

**Pull request merged**